src/v2/view-builder/views/password/EnrollAuthenticatorPasswordView.js

```diff
@@ -73,6 +73,15 @@ const Body = BaseForm.extend({
         }
       }
     ]);
+  },
+
+  saveForm() {
+    this.settings.processCreds({
+      username: this.options.appState.get('user')?.profile?.login,
+      password: this.model.get('credentials.passcode')
+    }).then(() => {
+      BaseForm.prototype.saveForm.apply(this, arguments);
```

Comment:
arrow functions have no `arguments`

Reply:
I just looked into this, it looks like the arguments variable works fine here. Arrow function's don't set the `arguments` variable at all, which means it get the `arguments` value from the outer scope. I set a breakpoint to double check. The existing device fingerprinting logic takes the same approach.